### ISSUE TITLE
"npm install" command breaks installation when previously installed 

### ISSUE THREAD

**Issue by prada-fu** [Teradata/nodejs-driver#2]
I've installed the nodejs-driver on Windows 10 (and Windows Server 2012) using the npm command "npm install teradata-nodejs-driver". If I later run the command "npm install" the node_modules\teradata-nodejs-driver\node_modules\ directory is deleted, but everything else in the node_modules\teradata-nodejs-driver\ directory remains. It looks like the same happens when running "npm ci" or "npm update". Re-running the original command "npm install teradata-nodejs-driver" fixes the issue, as does uninstalling and re-installing the package. Without the directory mentioned above, this error is received when creating a teradata connection object:
```
Error: Cannot find module '@teradataprebuilt/fastcall-win32-node-v64'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    ...
```

Also, potentially related, is that the install command won't run if I use optional parameters. For example "npm install teradata-nodejs-driver --loglevel verbose" fails due to "--loglevel verbose" being passed as optional parameters to node_modules\teradata-nodejs-driver\scripts\install-dependencies.js.

**Comment by jeremysmartt** [comment]
@prada-fu Thank you for the issue.  Will take a look 👍

**Comment by leogariba** [comment]
Same. 

```
internal/modules/cjs/loader.js:589
    throw err;
    ^

Error: Cannot find module '@teradataprebuilt/fastcall-linux-node-v67'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:587:15)
    at Function.Module._load (internal/modules/cjs/loader.js:513:25)
    at Module.require (internal/modules/cjs/loader.js:643:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    ...
```

Fedora 30
Node 11.0.0
NPM 6.4.1

**Comment by jeremysmartt** [comment]
Hello this has been fixed in the new release, `1.0.0-beta.2`
